metrics-operator/chart/values.yaml

```diff
@@ -62,6 +87,14 @@ replicas: 1
 tolerations: [ ]
 ## @param    topologySpreadConstraints add custom topology constraints to metrics operator
 topologySpreadConstraints: [ ]
+## @param     annotations add deployment level annotations
+annotations: {}
+## @param     podAnnotations adds pod level annotations
+podAnnotations: {}
+## @param  schedulingGatesEnabled enables the scheduling gates in lifecycle-operator. This feature is available in alpha version from K8s 1.27 or 1.26 enabling the alpha version
+schedulingGatesEnabled: false
```

Comment:
please correct me if i am amiss, but also here i think we might not need the `schedulingGatesEnabled` parameter

Reply:
yes I was supposed to copy something else here 😢